src/index.js

```diff
@@ -183,6 +183,14 @@ export default class RNPickerSelect extends PureComponent {
     }
 
     togglePicker(animate = false) {
+        if (this.state.showPicker) {
+            this.props.onBlur()
+        }
+        else {
```

Comment:
`} else {` :trollface:

Reply:
@gsteigerwald you out here reviewing faster than my actual team members 😂